lib/wakes/location_string.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module LocationString
```

Comment:
@chrismaddocks if you're going to go this route (which makes sense, since you need it in the `RedirectMapper` code too, I would probably rather go with a class. The module name here is pretty bad, and I think the difficulty ended up with a good name is because it needs to be a class.

I would probably do the following

    class URIFromLocationString
      def initialize(location_string)
        location_string = "http://#{location_string}" if URI(location_string).scheme.nil?
        URI(location_string)
      end
    end

Then you use it like `URIFromLocationString.new('/some/path').host`